cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -470,3 +470,13 @@ func (o *objectGraph) setCRSTenant(node, tenant *node) {
 		}
 	}
 }
+
+// checkVirtualNode logs if nodes are still virtual
+func (o *objectGraph) checkVirtualNode() {
+	log := logf.Log
+	for _, node := range o.uidToNode {
+		if node.virtual {
+			log.V(1).Info("Object won't be moved because it's not included in GVK considered for move", "kind", node.identity.Kind, "name", node.identity.Name)
```

Comment:
Before we merge this, trying to read through this message as a newcomer could be confusing, is there a better way to describe, or some links to documentation on how to debug?

Reply:
I guess we don't have a doc about this detail yet
should I write some info (with the information obtained above) then point it to there?

Reply:
As said in a previous comment, IMO this message is mostly for developers
Referece documentation for move/dicovery is https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#move

Reply:
An option is to change log level to v(5) so this is is debugging information as per https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/clusterctl/log/doc.go